specviz/data/qt/ui/equation_editor.ui

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like a duplicate? There is another one: specviz/plugins/arithmetic/equation_editor.ui and specviz/data/ui/equation_editor.ui

Reply:
Yeah, this needs to go! Was useful here when doing early dev work!